spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -197,32 +201,21 @@ public DeviceResult run(AndroidDebugBridge adb) {
     // Create the output directory, if it does not already exist.
     work.mkdirs();
 
-    LogRecordingTestRunListener recorder = new LogRecordingTestRunListener();
+    // determine the test set that is applicable for this device
```

Comment:
nit: capitalize Determine and put a period at the end. [Comments should be sentences.](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).